specification.md

```diff
@@ -1622,8 +1622,7 @@ to the trigger/produced event.
 
 | Parameter | Description | Type | Required |
 | --- | --- | --- | --- |
-| error | Domain-specific error name, or '*' to indicate all possible errors | string | yes |
-| code | Error code. Can be used in addition to the name to help runtimes resolve to technical errors/exceptions. Should not be defined if error is set to '*' | string | no |
+| condition | jq expression to resolve against input to error-handling logic | string | yes |
```

Comment:
what input? imo if we do this we need to describe that the mentioned "exceptions" json array needs to be injected by runtimes into the state data